### PR TITLE
API: Images sample code for GET request

### PR DESCRIPTION
**Pull request by martyav**
Sample code for the API documentation project. 

You can view my sandbox here: [https://www.mediawiki.org/wiki/User:Martyav/Sandbox/API:Images](https://www.mediawiki.org/wiki/User:Martyav/Sandbox/API:Images)

### CODE REVIEW

File: get_images.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/python3
+
+"""
+    images.py
```
Comment (srish) [inline get_images.py]: You could name this as "get_page_images.py"

File: get_images.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+    MediaWiki Action API Code Samples
+    Demo of `Images` module: Sending GET request to images
```
Comment (srish) [inline get_images.py]: You could elaborate the description a bit, something like "Send get request to obtain images embedded on a page.."

File: get_images.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    MIT license
+"""
+# Note: This code relies on the third party library, requests.
```
Comment (srish) [inline get_images.py]: Nice that you've added this statement here, but may be let's stick to our hello world template [here](https://www.mediawiki.org/wiki/API:Documentation_template#Sample_code) :-)

File: get_images.py [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+"""
+# Note: This code relies on the third party library, requests. 
+# Be sure to install it via your favorite package manager!
+import requests
+
+S = requests.Session()
+
+URL = "https://en.wikipedia.org/w/api.php"
+
+# Send a GET request to the images endpoint.
```
Comment (srish) [inline get_images.py]: You could also remove this line here.

File: get_images.py (hunk 1 above)
Comment (martyav) [reply get_images.py]: Changed name on file and in comment with latest commit

File: get_images.py (hunk 2 above)
Comment (martyav) [reply get_images.py]: Text now reads 

> Demo of `Images` module: Send a GET request to obtain a JSON
    object listing all of the image files embedded on a single
    page.

in renamed file

File: get_images.py (hunk 3 above)
Comment (martyav) [reply get_images.py]: Removed from renamed file

File: get_images.py (hunk 4 above)
Comment (martyav) [reply get_images.py]: Removed comment from renamed file

**Comment by srish** [comment]
Sweet! :-)

**Pull request merged**